saleor/permission/utils.py

```diff
@@ -32,9 +32,7 @@ def one_of_permissions_or_auth_filter_required(
     requestor = get_user_or_app_from_context(context)
 
     if requestor and permissions:
-        perm_checks_results = []
-        for permission in permissions:
-            perm_checks_results.append(requestor.has_perm(permission))
+        perm_checks_results = [requestor.has_perm(perm) for perm in permissions]
```

Comment:
This variable could be a generator. Passing it to `any` would result in stopping the iteration on the first `True` returned by it.

Reply:
> Current implementation also checking the exact same thing. Checking any permission from the list after the iteration and here we are doing the same thing with list comprehension.